docs/source/getting_started.rst

```diff
@@ -10,19 +10,21 @@ Installing FEniCS
 
 FESTIM requires FEniCS to run.
 
-It can be installed using `Docker <https://www.docker.com/>`_::
+The FEniCS project provides a prebuilt Anaconda python package (Linux and MacOS only) with FEniCS and its dependencies (python3, UFL, DOLFIN, numpy, sympy...)  already installed. 
+Anaconda can also be used in Windows using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) with your favourite linux distro. 
+First [install Anaconda](https://docs.continuum.io/anaconda/install) then run the following commands 
+
+    conda create -n festim-env -c conda-forge fenics
+    source activate festim-env
```

Comment:
Should we recommend using mamba for a faster installation?

```suggestion
    conda create -n festim-env
    source activate festim-env
    conda install mamba
    mamba install -c conda-forge fenics
```

Reply:
yeah for sure, it does take a while